File: 50 Coding Challenge Part I/Python/Challenge4.py
"""
Challenge 4: Print all the multiplication tables with numbers from 1 to 10
"""
num = 1
while num <= 10:
    print(f"Multiplication table for {num}")
    for i in range(1, 11):
        print(f" {num} * {i} = {num * i}")
    num += 1

File: 50 Coding Challenge Part I/C/challenge4.c
// Coding challenge 4: Print the multiplications tables with numbers from 1 to 10

#include <stdio.h>

int main(int argc, char *argv[]) {

	for (int i = 1; i <= 10; ++i) {

		printf("Multiplication table of %d\n", i);
		for (int j = 1; j <= 10; ++j) {
			printf("%d * %d = %d\n", j, i, i*j);
		}
	}

	return 0;
}
